khl/client.py

```diff
@@ -363,3 +367,28 @@ async def fetch_blocked_friends(self) -> List[Friend]:
 
     async def start(self):
         await asyncio.gather(self.handle_pkg(), self.gate.run(self._pkg_queue))
+
+    async def message_view(self, msg_id: str) -> Dict:
```

Comment:
naming: method name would be better if it is a verb/verb phrase since can convey this is a action/method
```suggestion
    async def view_message(self, msg_id: str) -> Dict:
```